scripts/build_check.sh

```diff
@@ -6,12 +6,11 @@ pipenv run pydocstyle .
 pipenv run mypy .
 mdl .
 
-# We use nmap to check if dynamodb is running locally
-# TODO: later should combine this with environmental variable checking (to see
-# if we can use a remote server instead).
-# XXX: Find a better way to check if dynamodb is running locally
+# We use a script to check if dynamodb is running locally
 COV_OPTIONS="--mypy --cov=./ --cov-branch --cov-config .coverageac"
-if nmap localhost | egrep "8000.*http-alt"; then
+PORT_BUSY="pipenv run python scripts/port_busy.py 8000"
```

Comment:
Might be worthwhile to make this relative to the repo root dir (i.e. with an env var) in case people don't run it from there.